CI issue with test_sdist-3.8_glx-win

***In GitLab by @woutdenolf on Jan 14, 2024, 12:06 GMT+1:***

https://gitlab.esrf.fr/workflow/ewoks/ewoks/-/jobs/914495

Regression in pytest 8.0.0rc1 (7.4.4 was the last working version).

```
___ ERROR collecting build_venv/Lib/site-packages/ewokscore/tests/examples ____
..\build_venv\lib\site-packages\_pytest\runner.py:341: in from_call
    result: Optional[TResult] = func()
..\build_venv\lib\site-packages\_pytest\runner.py:380: in collect
    collector.config.pluginmanager._loadconftestmodules(
..\build_venv\lib\site-packages\_pytest\config\__init__.py:616: in _loadconftestmodules
    mod = self._importconftest(conftestpath, importmode, rootpath)
..\build_venv\lib\site-packages\_pytest\config\__init__.py:662: in _importconftest
    assert mod not in mods
E   AssertionError
_____ ERROR collecting build_venv/Lib/site-packages/ewokscore/tests/utils _____
..\build_venv\lib\site-packages\_pytest\runner.py:341: in from_call
    result: Optional[TResult] = func()
..\build_venv\lib\site-packages\_pytest\runner.py:380: in collect
    collector.config.pluginmanager._loadconftestmodules(
..\build_venv\lib\site-packages\_pytest\config\__init__.py:616: in _loadconftestmodules
    mod = self._importconftest(conftestpath, importmode, rootpath)
..\build_venv\lib\site-packages\_pytest\config\__init__.py:662: in _importconftest
    assert mod not in mods
E   AssertionError
```

Probably `pytest` issue https://redirect.github.com/pytest-dev/pytest/issues/9765 (fixed in https://redirect.github.com/pytest-dev/pytest/pull/11708)

**Assignees:** @woutdenolf

*Migrated from GitLab: https://gitlab.esrf.fr/workflow/ewoks/ewoks/-/issues/23*

--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -30,7 +30,7 @@ test_sdist-3.8_glx-win:
   extends: .test_sdist-3.8_glx-win
   script:
     - !reference [.test_sdist-3.8_glx-win, script]
-    - python -m pip install ewokscore[test] "pytest==7.4.4"
+    - python -m pip install ewokscore[test]
     - python -m pytest -v --pyargs ewokscore.tests
     - python -m pytest -v --pyargs ewoksppf.tests
     - python -m pytest -v --pyargs ewoksdask.tests

